tox.ini

```diff
@@ -0,0 +1,47 @@
+[tox]
+envlist = py, flake8
+
+
+[testenv]
+usedevelop = True
+deps =
+    -rrequirements.txt
+    -rrequirements-dev.txt
+# setenv =
+#    PYTHONPATH = {toxinidir}
+commands = pytest . {posargs}
+
+[testenv:flake8]
+skip_install = true
+deps =
+    flake8
+    flake8-docstrings
+    flake8-import-order
+    pep8-naming
+    flake8-colors
```

Comment:
Not sure i want all of the errors these are spitting out to fail, some of them I fundamentally disagree with.